scripts/example.R

```diff
@@ -4,4 +4,13 @@ example_fxn <- function(){
   x <- "supercalifragilisticexpialidocious"
   return(x)
   
+
+  
+}
+
+
```

Comment:
Well done! 

Could you add some comments to document your code? If you are looking for an example of function documentation checkout this [example](https://github.com/kdorheim/history4hector/blob/5adf62bf55b860cdae0d0d6e4b0ed634bd1d187d/scripts/L0.CEDS.R#L9)